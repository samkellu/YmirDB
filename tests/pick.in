pick a 1
set a 123 44 55
pick a 5
pick a 1
set b a 23
pick b 1
bye
