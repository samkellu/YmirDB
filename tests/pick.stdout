==810== Memcheck, a memory error detector
==810== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==810== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==810== Command: ./ymirdb
==810== 
==810== error calling PR_SET_PTRACER, vgdb might block
> no such key

> ok

> index out of range

> 123

> ok

> a

> bye
==810== 
==810== HEAP SUMMARY:
==810==     in use at exit: 0 bytes in 0 blocks
==810==   total heap usage: 38 allocs, 38 frees, 2,024 bytes allocated
==810== 
==810== All heap blocks were freed -- no leaks are possible
==810== 
==810== For lists of detected and suppressed errors, rerun with: -s
==810== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
